message.go

```diff
@@ -10,6 +10,8 @@ import (
 type msgType int
 
 const (
+	// Whisper
+        WHISPER msgType = 0
```

Comment:
Just move this under ROOMSTATE like this:

```
const (
	// PRIVMSG standard chat message
	PRIVMSG msgType = iota + 1
	// CLEARCHAT timeout messages
	CLEARCHAT
	// ROOMSTATE changes like sub mode
	ROOMSTATE
	// WHISPER private user targeted whispers
	WHISPER
)
```

Reply:
My concern with changing this is that message type ID for outgoing whisper is WHISPER is 0 (which is what I'm mainly using this for), and incoming whisper is 4 (not 100% confirmed)

Reply:
Okay. Hmm this was actually not intended to be used that way but I see why you are using it. It's okay to merge im gonna think how to refactor that part later